python/python/glide/async_commands/core.py

```diff
@@ -6002,6 +6003,115 @@ async def watch(self, keys: List[TEncodable]) -> TOK:
             await self._execute_command(RequestType.Watch, keys),
         )
 
+    async def xclaim(
```

Comment:
would you please move this command so it's right _before_ xautoclaim (line 3193)

Reply:
addressed